x-pack/legacy/plugins/ml/public/jobs/new_job_new/pages/components/pick_fields_step/components/advanced_detector_modal/advanced_detector_modal.tsx

```diff
@@ -360,7 +360,8 @@ function createFieldOptionList(agg: Aggregation | null) {
     .map(createFieldOption);
 
   // working round EuiComboBox's odd behavior when the options list contains only one item
-  return options.length === 1 ? [emptyOption, ...options] : options;
+  // return options.length === 1 ? [emptyOption, ...options] : options;
```

Comment:
Can this comment go?

Reply:
yes, i had removed it but forgot to push the change